src/converters/piral-ng/src/types.ts

```diff
@@ -17,6 +20,8 @@ export interface NgComponent {
    * The type of the Angular component.
    */
   type: 'ng';
+
```

Comment:
Would be superb if you could place some descriptive comment on top. Thanks!

Reply:
NP, added doc comment.